packages/web/src/common/store/cache/collections/createPlaylistSaga.ts

```diff
@@ -114,7 +114,12 @@ function* optimisticallySavePlaylist(
   playlist.is_private = true
   playlist.playlist_contents = {
     track_ids: initTrack
-      ? [{ time: Date.now(), track: initTrack.track_id }]
+      ? [
+          {
+            time: Math.round(Date.now() / 1000),
```

Comment:
wow that's crazy

Reply:
Mind adding a comment to explain it? Just don't want future people examining it to be scratching their heads :-D